src/cmark.h

```diff
@@ -466,11 +466,24 @@ CMARK_EXPORT void cmark_consolidate_text_nodes(cmark_node *root);
 CMARK_EXPORT
 cmark_parser *cmark_parser_new(int options);
 
```

Comment:
Can we have a comment here in the same style as with the other exported functions?

Reply:
oh oops that was a stray function I never ended up making.

Did add more docs though.